sync/strategies/simple_onchange.py

```diff
@@ -0,0 +1,42 @@
+from sync.interfaces import IStrategy
+class SimpleOnChangeStrategy(IStrategy):
+    last_seen = {}
+
+    # A strategy that watches emulator memory & sends it when a change is detected
+    # and applies data immediately when it is received (before collecting data to send)
+    def __init__(self, offset_length_pairs):
+        self.offset_length_pairs = offset_length_pairs
+        self.interesting_offsets = list(list(zip(*offset_length_pairs))[0]) # just a list of all the offsets
```

Comment:
`self.interesting_offsets = [offset for offset, _ in offset_length_pairs]` is more readable than `list(list(zip(*))[0])`